openassessment/xblock/test/test_staff.py

```diff
@@ -283,25 +306,37 @@ def test_staff_assess_handler(self, xblock):
         self.submit_staff_assessment(xblock, submission, assessment=TEAM_GOOD_ASSESSMENT)
 
         assessment = teams_api.get_latest_staff_assessment(submission['team_submission_uuid'])
-        self._assert_team_assessment(assessment, submission)
+        self._assert_team_assessment(assessment, submission, self.expected_answer)
+
+    @scenario('data/team_submission.xml', user_id='Bob')
+    def test_staff_assess_handler_regrade(self, xblock):
+        submission = self._setup_xblock_and_create_team_submission(xblock)
```

Comment:
Not super obvious what this is doing. Adding a comment/docstring or logical grouping will make this read better.